omod/src/main/webapp/resources/scripts/order-entry.js

```diff
@@ -229,14 +229,19 @@ angular.module("orderEntry", ['orderService', 'encounterService', 'session'])
                     delete transformed.editing;
                     return transformed;
                 });
+                
                 var encounter = {
                     patient: encounterContext.patient.uuid,
                     encounterType: encounterContext.encounterType.uuid,
                     visit: uuidIfNotNull(encounterContext.visit),
                     location: uuidIfNotNull(encounterContext.location),
-                    provider: provider.person.uuid, // submit the person because of RESTWS-443
+                    // provider: provider.person.uuid, // submit the person because of RESTWS-443
                     orders: orders,
+					encounterProviders: [
+                		{ provider: provider.uuid, encounterRole: encounterContext.encounterRole.uuid }
+                	]
```

Comment:
@dkayiwa : Here is the code where I have replaced the provider to encounterProviders which has the provider and encounterRole.